code/game/objects/structures/fluff.dm

```diff
@@ -274,7 +274,7 @@
 	icon = 'icons/obj/tram_rails.dmi'
 	icon_state = "rail"
 	layer = TRAM_RAIL_LAYER
```

Comment:
change this value, layers are divided up by plane so this needs to go lower

Reply:
nvm the pr I did on wallening isn't here so floor layers are intermixed with with game plane layers